Libraries/LibGfx/VulkanContext.cpp

```diff
@@ -75,14 +75,19 @@ static ErrorOr<VkDevice> create_logical_device(VkPhysicalDevice physical_device)
     queue_families.resize(queue_family_count);
     vkGetPhysicalDeviceQueueFamilyProperties(physical_device, &queue_family_count, queue_families.data());
 
-    int graphics_queue_family_index = -1;
-    for (int i = 0; i < static_cast<int>(queue_families.size()); i++) {
+    bool graphics_queue_family_found = false;
+    uint32_t graphics_queue_family_index = 0;
+    for (auto& i = graphics_queue_family_index; i < queue_families.size(); ++i) {
```

Comment:
it doesn't seem like `auto&` is intentional here. did you mean to use just `auto`?

Reply:
Definitely the auto& is intentional.  Variables 'i' and 'graphics_queue_family_index' need to be the same value, so they represent the same stack variable.  I want to use the name 'i' because it is shorter for the loop, but I want the value available later on line 93 with the longer named 'graphics_queue_family_index'.

If this is not the clearest, and simplest, well I would agree. I could resubmit using only one variable name if this is a problem.

Reply:
we could make this loop increment `graphics_queue_family_index` and remove `i` completely.

Reply:
Sounds good. I'll do that.